src/components/InputField/index.tsx

```diff
@@ -21,14 +22,15 @@ const Input = ({
     error,
     disabled = false,
     className = '',
+    required = false,
 }: InputFieldProps) => (
     <div className={`form-group ${className}`}>
         {label && <label className='form-group__label wt-400'>{label}</label>}
         <FormControl
             type={type}
             value={value}
             onChange={(e) => onChange?.(e as React.ChangeEvent<HTMLInputElement>)}
-            placeholder={placeholder}
+            placeholder={`${placeholder} ${required ? '*' : ''}`}
```

Comment:
```suggestion
            placeholder={`${placeholder} ${required && '*'}`}
```

Reply:
this added `False` in the placeholder if it was not required